index.html

```diff
@@ -0,0 +1,42 @@
+<html>
+<head>
+  <title>Tic tac toe</title>
+  <link rel="stylesheet" href="style.css" media="screen" type="text/css" />
+
+</head>
+	<body onload="startGame()";>
```

Comment:
All javascript should live in your javascript file.  You can use the AddEventListener you learned about this week.